tests/unittests/tests-core/tests-core-xfa.h

```diff
@@ -27,6 +27,10 @@ typedef struct {
     const char *text;
 } xfatest_t;
 
+typedef struct __attribute__((aligned(4))) {
```

Comment:
Wouldn't

```suggestion
typedef struct WORD_ALIGNED {
```

from `#include "architecture.h"` be sufficient here?

Reply:
yup, fixed